docs/deploy/configuration/replica-movement.mdx

```diff
@@ -0,0 +1,324 @@
+      startMarker="# START CheckShardingState"
+      endMarker="# END CheckShardingState"
+      language="py"
+    />
+  </TabItem>
+  <TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+<TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+</Tabs>
+
+<details>
+  <summary>Code output</summary>
+
+```
+Shards in 'MyReplicatedDocCollection': ['BoMYGMWFncSG', 'ddYSP1oIHT7t', 'xSBcs4TRJ5QH']
+Nodes for shard 'BoMYGMWFncSG': ['node3', 'node1']
+```
+
+</details>
+
+## Initiating a replica movement operation
+
+This starts the asynchronous process of creating a replica on a new node and, if moving, removing it from the old one. When copying a replica, the replication factor for that shard will be increased as opposed to moving the replica. You'll need to specify the source node, destination node, collection name, shard ID and whether the operation is a `MOVE` or `COPY`.
+
+<Tabs groupId="languages">
+<TabItem value="py" label="Python Client v4">
+<FilteredTextBlock
+text={PyCode}
+startMarker="# START ReplicateShard"
+endMarker="# END ReplicateShard"
+language="py"
+/>
+</TabItem>
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+<TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+</Tabs>
+
+<details>
+  <summary>Code output</summary>
+
+```
+Replication initiated, ID: 32536c0e-09e1-4ea1-a2c5-e85af10a9d58
+```
+
+</details>
+
+## Checking the status of a replication operation
+
+Since movement is asynchronous, you can monitor its progress. The movement operation can have one of the following states:
+
+- `REGISTERED`
+- `HYDRATING`
+- `FINALIZING`
+- `DEHYDRATING`
+- `READY`
+- `CANCELLED`
+
+To learn more about the replication states, check out [Concepts: Cluster architecture](/docs/weaviate/concepts/replication-architecture/cluster-architecture.md#replica-movement).
+
+<Tabs groupId="languages">
+<TabItem value="py" label="Python Client v4">
+<FilteredTextBlock
+text={PyCode}
+startMarker="# START CheckOperationStatus"
+endMarker="# END CheckOperationStatus"
+language="py"
+/>
+</TabItem>
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+<TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+</Tabs>
+
+<details>
+  <summary>Code output</summary>
+
+```
+Status for 32536c0e-09e1-4ea1-a2c5-e85af10a9d58: REGISTERED
+```
+
+</details>
+
+## Listing replication operations
+
+List all ongoing or recently completed operations. This can be filtered by node, collection and shard.
```

Comment:
What's `recently completed` 🤔?

I think we need to be more specific. If we have more info, it could be added to references/concepts and linekd here imo